config/rollup.config.js

```diff
@@ -0,0 +1,78 @@
+import path from 'path';
+
+import makePlugins from './plugins/index';
+import makeDevServerPlugins from './plugins/server';
+
+const pkgInfo = require('../package.json');
+
+const isProduction = process.env.NODE_ENV === 'production';
+
+// default config that is used across all builds
+const config = {
+  input: pkgInfo.source || './index.js',
```

Comment:
`./index.js` is the example deck, so I'm not sure this is a good default (although for a different project would totally make sense). Maybe point to `./src/index.js`?